inference-engine/thirdparty/clDNN/tests/test_cases/extract_image_patches_gpu_test.cpp

```diff
@@ -0,0 +1,577 @@
+/*
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#include <gtest/gtest.h>
+
+#include <api/memory.hpp>
+#include <api/input_layout.hpp>
+#include <api/extract_image_patches.hpp>
+#include <api/topology.hpp>
+#include <api/network.hpp>
+#include <api/data.hpp>
+
+#include <test_utils/test_utils.h>
+
+using namespace cldnn;
+using namespace tests;
+
+TEST(extract_image_patches_gpu, basic) {
+    //  Input  : 1x1x10x10
+    //  Output : 1x9x2x2
+
+    tensor output_shape = {1, 9, 2, 2};
+    const auto& engine = get_test_engine();
+    auto batch = 1;
+    auto depth = 1;
+    auto in_rows = 10;
+    auto in_cols = 10;
+    std::vector<unsigned int> sizes = {3, 3};
+    std::vector<unsigned int> strides = {5, 5};
+    std::vector<unsigned int> rates = {1, 1};
+    std::string auto_pad = "valid";
+
+    auto input = memory::allocate(engine, { data_types::f32, format::bfyx, { batch, depth, in_rows, in_cols } });
```

Comment:
Note that tensor constructor is `tensor(value_type batch_num, value_type feature_num, value_type width, value_type height)`, so order `batch, depth, in_cols, in_rows` would be more appropriate

Reply:
Thank you. I've changed in other places and forgot here. I'll update.